app/src/renderer/scripts/utils.js

```diff
@@ -0,0 +1,14 @@
+import createHash from "create-hash"
```

Comment:
Couldn't we just get this from the Node crypto module to avoid adding a dependency?

Reply:
nice, didn't realize it was also that ez